DirectEmployers/MyJobs
# [PD 2666] styling of templates

To Test:
In the admin switch over to the home_page_listing_bootstrap3.html template for the home template with Version 2 selected.

Styles have been added for the rebranding along with some functionality for desktop and mobile. Need to check and make sure all of the pages have the styling included along with the left hand facets, footer, navigation, etc.
